contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,166 @@
+
+You might be wondering, "How can I do the same?"
+
+In this post we show you just that. We break down [Superhuman's product-market fit engine](https://review.firstround.com/how-superhuman-built-an-engine-to-find-product-market-fit) into an easy step-by-step guide you can follow as well as dive into two case studies.
+
+## A very brief introduction to PMF surveys
+
+> If you're already familiar with product-market fit surveys, skip to the [next section](#how-to-drive-product-market-fit-with-pmf-surveys).
+
+Created by entrepreneur Sean Ellis, the core question (among others) it asks is:
+
+"How would you feel if you could no longer use [product name]?"
+
+- a) Very disappointed
+- b) Somewhat disappointed
+- c) Not disappointed
+
+Your PMF score is the percentage of total responses that are "very disappointed".
+
+For example, let's say you survey `100` people. `50` people respond with "very disappointed", your PMF score would be '50 / 100 = 50%`.
+
+Based on his research of 100+ startups, Ellis believes 40% answering "very disappointed" is a strong signal of product-market fit.
+
+![How to calculate PMF score](../images/blog/pmf-surveys/pmf-score.png)
+
+## How to drive product-market fit with PMF surveys
+
+### Step 1: Survey informed users
+
+The first step is to learn more about your customers and what they think of your app. Do this by creating a [survey](/surveys) with the following questions:
+
+1. How would you feel if you could no longer use our product?
+   
+2. What type of people do you think would most benefit from it?
+
+3. What is the main benefit you receive from it?
+
+4. How can we improve our app for you?
+
+Each question plays an important and specific role – we'll explain how later. 
+
+In the meantime, to receive the best feedback, send the survey to users who have recently had meaningful interactions with your app. This means users that have:
+
+- experienced the core value-prop of your product.
+- used your product at least twice.
+- used your product in the last two weeks.
+
+You can begin analyzing the results once you have at least 30 responses. 
+
+### Step 2: Study your best customers
+
+Start by examining the users who would be "very disappointed" if they could no longer use your product. There are two things you need to learn about them:
+
+1. Who they are
+
+2. Why they love your product
+
+To find the answers, look at their responses to the questions `What is the main benefit you receive from our product?` and `What type of people do you think would most benefit from it?`.
+
+Then, use this information to create a [user persona](/product-engineers/how-to-create-user-personas) of your best supporters. Users who match this persona are the ones you need to improve your app for and you should focus your attention solely on them. In the next step we explain how to do this.
```

Comment:
> Users who match this persona are the ones you need to improve your app for and you should focus your attention solely on them.

It's not clear why. Maybe so you can improve survey results which mean your overall product is improving? Trickle down improvements to non-best supporters?